Formula/notmuch.rb

```diff
@@ -28,14 +27,12 @@ def install
     args = %W[
       --prefix=#{prefix}
       --mandir=#{man}
-      --with-emacs
```

Comment:
@gpanders What was this line doing? Was it just ignored?

Reply:
It tells `notmuch` to install its elisp files. It is enabled by default, so it's not necessary to include it.